README.rst

```diff
@@ -74,6 +74,7 @@ in development
 ^^^^^^^^^^^^^^
 * Smarter line wrapping in summary and parameters tables.
 * Any code inside of ``if __name__ == '__main__'`` is now excluded from the documentation.
+* Fixed that modules were processed in a random order leading to several hard to reproduce bugs.
```

Comment:
Minor comment
I don't understand this release note.
The release note is designed for end users to know whether they should upgrade or not.

---------

Maybe something like this

* The internal processing of modules was updated to an algorithm used fixed order. This should help fix some of the bugs that previously seemed to happen for no apparent reason.

Or just keep it internal and don't have a release note.